packages/pwa-kit-extension-sdk/src/configs/webpack/overrides-resolver-loader.ts

```diff
@@ -5,21 +5,33 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 import {LoaderContext} from 'webpack'
+import fse from 'fs-extra'
 import os from 'os'
 import path from 'path'
 import resolve from 'resolve'
 
 // Local Imports
-import {buildCandidatePaths, getPackageName} from '../../shared/utils'
+import {buildCandidatePaths, getPackageName, SETUP_FILE_REGEX} from '../../shared/utils'
 
 // Types
 import type {ExtendedCompiler} from './types'
 
 // Constants
+const EXTENSION_PACKAGE_PREFIX = 'extension-'
+const EXTENSION_PACKAGE_WORKSPACE = '@salesforce'
 const IMPORT_REGEX = /import\s+(?:(?:[\w*\s{},]*)\s+from\s+)?['"](\..*?)['"]/g
+const OVERRIDABLE_FILE_NAME = '.force_overrides'
+const MONO_REPO_WORKSPACE_FOLDER = `${path.sep}packages${path.sep}`
+const NODE_MODULES_FOLDER = `${path.sep}node_modules${path.sep}`
```

Comment:
(nitpicking)

it's slightly weird to see a path segment contain both leading and trailing slash, it makes the concatenation harder, I'd imagine most cases you want to pick one, either leading or trailing slash, but not both.

Reply:
I agree and was thinking the same thing.. but it also looked weird in the place where I was using it lol ... I'll look to find something that works out better

Reply:
Done